transforms/angle-brackets/test.js

```diff
@@ -888,3 +888,19 @@ test('custom-options', () => {
       "
   `);
 });
+
+test('preserve arguments', () => {
+  let input = `
+    {{foo-bar class="baz"}}
+    {{foo-bar data-baz class="baz"}}
+    {{link-to (t "show") "flight" event.flight.id class="btn btn-default btn-sm pull-right"}}
+  `;
+
+  expect(runTest('preserve-arguments.hbs', input)).toMatchInlineSnapshot(`
+    "
+        <FooBar @class=\\"baz\\" />
+        <FooBar data-baz @class=\\"baz\\" />
```

Comment:
I'm not sure about data attributes. I think we should convert them to `@data-baz` too except for built-in components. 🤔

Reply:
GRR!  I debated this too!

Reply:
Let me know what you want to do

Reply:
ya, everything should be named arguments

Reply:
easy.  Making the change